### PR TITLE
bcl2fastq-1.8.4-goolf-1.4.10.eb

### PR DESCRIPTION
**Pull request by pescobar**
this needs more testing

@pforai commented that this tool can be linked with openblas...I need to figure out how to do it...

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 0 out of 2
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/fa6fbeb6353adef39c71 for a full test report.

**Comment by rjeschmi** [comment]
I think this is an interesting thing to merge, but it has yet another version of boost for goolf-1.4.10

Does it need to?

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by rjeschmi** [comment]
I'm looking at this, but blocked on a Tk X11 dep further down.

**Comment by boegel** [comment]
we have Tk with `-no-X11` as of recent...

**Comment by rjeschmi** [comment]
@boegel yeah, that one doesn't work :)

File: easybuild/easyconfigs/b/bcl2fastq/bcl2fastq-1.8.4-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+# configure script has to be run in a different folder so we skip the configure step 
+# and will run configure in premakeopts
+skipsteps = ['configure']
+
+# the software requires to be built outside of the original source folder
+separate_build_dir = True
+
+# build system requires BOOST_ROOT env var to find the right boost library
+premakeopts = 'export BOOST_ROOT=$EBROOTBOOST && '
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/b/bcl2fastq/bcl2fastq-1.8.4-goolf-1.4.10.eb]: prebuildopts

File: easybuild/easyconfigs/b/bcl2fastq/bcl2fastq-1.8.4-goolf-1.4.10.eb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+# here we run ./configure
+premakeopts += '%(builddir)s/bcl2fastq/src/configure --prefix=%(installdir)s && '
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/b/bcl2fastq/bcl2fastq-1.8.4-goolf-1.4.10.eb]: ditto

File: easybuild/easyconfigs/b/bcl2fastq/bcl2fastq-1.8.4-goolf-1.4.10.eb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                                    "configureQseqToFastq.pl", "configureValidation.pl"]],
+    'dirs': ["etc", "lib", "libexec", "share"],
+}
+
+modextrapaths = {
+    'PERL5LIB': 'lib/bcl2fastq-1.8.4/perl'
+    }
+
+moduleclass = 'bio'
+
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/b/bcl2fastq/bcl2fastq-1.8.4-goolf-1.4.10.eb]: remove last empty line.

this is not an endorsement of Boost in any kind of way ;-)

lgtm , minor changes needed here, `easyblock = 'ConfigureMake'` , put this in REVIEW

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months, obsoleted by more recent PRs for newer versions.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**